x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -1119,6 +1287,9 @@ export const tasks: TelemetryTask[] = [
                 },
               },
             },
+            metricset: {
+              ...dataStreamStatsDictionary,
+            },
           },
           traces: {
             shards: {
```

Comment:
@kpatticha We talked about splitting up the tasks file. Is that something you are planning to work or will you create an issue? Thanks!

Reply:
This file definitely needs to splitted up. +1 for that.

Reply:
I created an issue https://github.com/elastic/kibana/issues/167234